awsapilib/console/console.py

```diff
@@ -204,25 +173,37 @@ def get_billing_root_session(self, redirect_url, unfiltered_session=False):
         if not self._get_console_root_session(redirect_url):
             raise InvalidAuthentication('Unable to get a valid authenticated session for root console.')
         service = 'billing'
-        url = f'{self.urls.billing_home}?#/account'
-        _ = self._session.get(url)
-        url = f'{self.urls.billing_home}'
+        url = self.urls.billing_home
```

Comment:
Maybe not use the intermidiate url variable here? the next call is pretty simple.